kenichi127/PoE
# User/admin home

## Description / Purpose 
It is a homepage of the user and the manager
Search, add data (admin only), send food list, send mail, search shelters
Fixes issue:https://docs.google.com/spreadsheets/d/1BHy8K_EvpG4fBLjujxShiZWe2-tNs0Jj/edit#gid=905621989
task:6
# # To test this PR
(all)
1.Create two user
2. The first one created is the administrator, the second and subsequent ones become the users.
(Search)
1. Select the desired date, time and place.
2. Click Search button.
3. The answer page will appear
(Add data (administrator only))
1. Click on the add button
2. A page for adding will appear
(Contribution)
1.Click the button of contribution.
2.Contribution page will appear.
(Food list)
1. Click the button of the food list.
2. Food list page comes out
(send the mail)
1. Click the Send Mail button
2. Email will be sent. (Return to home)
(shelter search)
1. Click on the shelter button
2. Search for shelters on Googlemap
** Command **
php artisan migrate:refresh

 **screenshot**
(admin home)
![image](https://user-images.githubusercontent.com/49220772/58773405-882f7400-85ef-11e9-89ee-61e3837c4f34.png)
(additional page)
![image](https://user-images.githubusercontent.com/49220772/58773447-aa28f680-85ef-11e9-8cf9-e106a4399754.png)
(user home)
![image](https://user-images.githubusercontent.com/49220772/58773519-e5c3c080-85ef-11e9-865f-44bb3a8dc98e.png)
(Search page)
![image](https://user-images.githubusercontent.com/49220772/58773536-fb38ea80-85ef-11e9-83e4-3dd9b5074f07.png)
(foodlist page)
![image](https://user-images.githubusercontent.com/49220772/58773593-31766a00-85f0-11e9-9f5d-faceffcfc067.png)
(Contribution Page)
![image](https://user-images.githubusercontent.com/49220772/58773629-5b2f9100-85f0-11e9-97f9-56a9fc825f20.png)
(shelter searching)
![image](https://user-images.githubusercontent.com/49220772/58773646-74384200-85f0-11e9-86b5-4fe66933c031.png)
